monai/transforms/intensity/array.py

```diff
@@ -876,7 +1069,8 @@ def __call__(self, img: NdarrayOrTensor, randomize: bool = True) -> NdarrayOrTen
 
         if self.gamma_value is None:
             raise RuntimeError("gamma_value is not set, please call `randomize` function first.")
-        return AdjustContrast(self.gamma_value)(img)
+
+        return AdjustContrast(self.gamma_value, invert_image=self.invert_image, retain_stats=self.retain_stats)(img)
```

Comment:
I realise this pattern was in the original code so that's on us, but could we also make this object a member attribute? The same for elsewhere this occurs.

Reply:
Agreed. This is now instantiated in the constructor as suggested.